### DISCUSSION TITLE
Multiple gopro as webcam simultaneously over USB

### DISCUSSION THREAD

**Discussion by mehrajagdish** [gopro/OpenGoPro#284] [Q&A]
Is there a way to get multiple gopro's live feed over USB. I want to use gopro feed for my opencv project but can't find a way to access both camera simultaneously .

**Comment by DDANGEUN** [comment]
use udp://@172.2X.1YZ.5**2**:8554
you can see the stream separately for each GoPro IP. (check cmd>ipconfig)
I worte about this a few days ago. https://github.com/gopro/OpenGoPro/discussions/282
you're so lucky. save your time.

**Reply by mehrajagdish** [reply]
Is there any way i can access the camera feed without using UDP stream and directly via USB.

**Reply by KonradIT** [reply]
Nope. Ingest each UDP stream programatically, forget about using the GoPro Webcam software.
